Formula/lua.rb

```diff
@@ -109,82 +83,72 @@ def install
   end
 
   def pc_file; <<~EOS
-    V= 5.2
-    R= 5.2.4
-    prefix=#{opt_prefix}
+    V= 5.3
+    R= #{version}
+    prefix=#{HOMEBREW_PREFIX}
     INSTALL_BIN= ${prefix}/bin
     INSTALL_INC= ${prefix}/include
     INSTALL_LIB= ${prefix}/lib
     INSTALL_MAN= ${prefix}/share/man/man1
-    INSTALL_LMOD= #{HOMEBREW_PREFIX}/share/lua/${V}
-    INSTALL_CMOD= #{HOMEBREW_PREFIX}/lib/lua/${V}
+    INSTALL_LMOD= ${prefix}/share/lua/${V}
+    INSTALL_CMOD= ${prefix}/lib/lua/${V}
     exec_prefix=${prefix}
     libdir=${exec_prefix}/lib
     includedir=${prefix}/include
 
     Name: Lua
     Description: An Extensible Extension Language
-    Version: 5.2.4
+    Version: #{version}
     Requires:
-    Libs: -L${libdir} -llua -lm
+    Libs: -L${libdir} -llua.5.3 -lm
     Cflags: -I${includedir}
     EOS
   end
 
-  def caveats; <<~EOS
```

Comment:
This caveat is still relevant but the same thing happens with `python` and `python3` and we don't warn about it so I don't have strong feelings 🤷‍♂️.